src/pudl/metadata/resources/ferc1_eia_record_linkage.py

```diff
@@ -124,6 +127,34 @@
         "etl_group": "outputs",
         "field_namespace": "eia",
     },
+    "out_eia__yearly_assn_plant_parts_plant_gen": {
```

Comment:
Is it clear whether this should be an `out_eia__` or an `out_pudl__` table?  The FERC-EIA plant parts association is currently `out_pudl__` but the plant parts table itself is `out_eia__`.  The fields on both sides of the association here are ultimately somehow derived from EIA, but the association itself feels like a PUDL construction.

```
out_eia__yearly_plant_parts
out_pudl__yearly_assn_eia_ferc1_plant_parts
```